build.json

```diff
@@ -39,11 +44,6 @@
       "ol.ENABLE_DOM=false",
       "ol.ENABLE_WEBGL=false"
     ],
-    "js": [
-      "node_modules/openlayers/externs/olx.js",
-      "node_modules/openlayers/externs/oli.js",
-      "node_modules/ngeo/externs/ngeox.js"
```

Comment:
OK, I wasn't aware of that. I made the change because I had noticed it wasn't the case in agridea: https://github.com/camptocamp/agridea_geoacorda/blob/master/jsapi-buildtools/geoacorda.json
I will revert the change.